openfga/v2beta1/openfga_service.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package openfga.v2beta1;
+
+import "google/protobuf/struct.proto";
+import "openfga/v2beta1/openfga.proto";
+
+service OpenFGAService {
+    rpc Check(CheckRequest) returns (CheckResponse);
+    rpc ListObjects(ListObjectsRequest) returns (ListObjectsResponse);
+
+    rpc DeleteRelationships(DeleteRelationshipsRequest) returns (DeleteRelationshipsResponse);
+    rpc WriteRelationships(WriteRelationshipsRequest) returns (WriteRelationshipsResponse);
+
+    rpc WriteModel(WriteModelRequest) returns (WriteModelResponse);
+    rpc ReadModel(ReadModelRequest) returns (ReadModelResponse);
+    rpc ListModels(ListModelsRequest) returns (ListModelsResponse);
+}
+
+message CheckRequest{
+    string store_id = 1;
+    string model_id = 2;
+    Subject subject = 3;
+    string relation = 4;
+    Object object = 5;
+    repeated RelationshipTuple contextual_tuples = 6;
+    google.protobuf.Struct context = 7;
+}
+
+message CheckResponse{}
+
+message ListObjectsRequest{
+    string store_id = 1;
+    string model_id = 2;
+    openfga.v2beta1.Subject subject = 3;
+    string relation = 4;
+    string object_type = 5;
```

Comment:
For consistency, maybe:
object: {
  type: ...
}

Reply:
That would imply another message definition of the form

```
message ObjectWithoutId {
    string type = 1;
}
```

That's a bit overloaded IMO. `string object_type` points to the `Object.Type` field. I think accepting the string is more straightforward.